components/ImageInBrowserFrame.tsx

```diff
@@ -20,7 +20,7 @@ export default function ImageInBrowserFrame({
     <div
       style={{
         padding: "20px 0 0",
-        margin: 10,
```

Comment:
This change ensures that there's enough margin for the drop shadow around the screenshot pages so that we don't need to add 2 `<br>` after them. Even though I fixed it in just the Hubspot doc, after we merge, we should clean up the other pages.

Reply:
Oh good. Yes, I didn't like it if the text landed in the shadow. I know the integration docs have lots of double <br> going on and I can clean those up.